source/globalCommands.py

```diff
@@ -113,6 +113,9 @@
 #: Script category for document formatting commands.
 # Translators: The name of a category of NVDA commands.
 SCRCAT_DOCUMENTFORMATTING = _("Document formatting")
+#: Script category for audio streaming commands.
+# Translators: The name of a category of NVDA commands.
+SCRCAT_AUDIOSTREAMING = _("Audio streaming")

```

Comment:
Isn't streaming too technical to be understandable? I'd certainly have a difficult time if I had to translate it to Polish. Perhaps something like 'audio managementt' will be better?